apps/new/widget/home/Footer.jsx

```diff
@@ -0,0 +1,307 @@
+const { Button } = VM.require("buildhub.near/widget/components") || {
+  Button: () => <></>,
+};
+const Logo =
+  "https://ipfs.near.social/ipfs/bafkreihsuyli6i2wphsutag6xcxnyhyrn7wtkklvqebx4szgpz3ieqacxu";
+
```

Comment:
let's make a assets folder for refactoring and move all image and svgs there, it will help cleanup the code
please create a ticket for it